tests/e2e/script_test.go

```diff
@@ -78,8 +87,11 @@ func TestE2EScript(t *testing.T) {
 				var eventsByStep [][]*test.LogEntry
 
 				eventsBefore := h.Events.HTTPEvents
-				for i, obj := range script.Objects {
+				// tracks the set of all applied objects as keyed by a gvk, namespaced name tuple
+				appliedObjects := map[gvkByNN]*unstructured.Unstructured{}
+				ignoreStepEvents := map[int]struct{}{} // ignore steps for which we don't records events for like CC/ CCC
```

Comment:
I don't think I understand why we need to skip this; if we write an empty file we are asserting that there are no events... We would skip if there was unavoidable non-determinism...